File: SourceAFIS/Matching/Minutia/EdgePair.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SourceAFIS.Matching.Minutia
{
    public struct EdgePair
    {
        public MinutiaPair Reference;
        public MinutiaPair Neighbor;

        public EdgePair(MinutiaPair reference, MinutiaPair neighbor)
        {
            Reference = reference;
            Neighbor = neighbor;
        }
    }
}

File: java/sourceafis/src/sourceafis/matching/minutia/EdgePair.java
package sourceafis.matching.minutia;
 public class EdgePair
    {
        public MinutiaPair reference;
        public MinutiaPair neighbor;

        public EdgePair(MinutiaPair reference, MinutiaPair neighbor)
        {
            this.reference = reference;
            this.neighbor = neighbor;
        }
    }